vllm_spyre/v1/worker/spyre_input_batch.py

```diff
@@ -448,11 +471,11 @@ def get_unpadded_output_indices(self) -> dict[str, int]:
         This will output: {"B": 0, "C": 1}
         """
 
-        indices = self.model_indices_mask.nonzero().squeeze(dim=-1).tolist()
+        indices = self.req_indices_mask.nonzero().squeeze(dim=-1).tolist()
```

Comment:
can we change the docstring for this function to reflect `self.req_indices_mask` as well?